resources/idlharness.js

```diff
@@ -268,7 +267,7 @@ IdlArray.prototype.add_dependency_idls = function(raw_idls, options)
         let name = parsed.name
             || parsed.type == "implements" && parsed.target
             || parsed.type == "includes" && parsed.target;
-        if (skip(name) || !all_deps.has(name)) {
+        if (!name || should_skip(name) || !all_deps.has(name)) {
```

Comment:
Still can't quite tell why it's possible for `name` to be null. Is it `parsed.target` which can be null?

Reply:
If the chunked is unnamed and doesn't implement/include. Can't recall the case that isn't any of these, but I definitely hit it in my testing.